docs/src/main/paradox/migration-guide/migration-guide-10.2.x.md

```diff
@@ -0,0 +1,14 @@
+# Migration Guide to and within Akka HTTP 10.2.x
+
+## General Notes
+
+See the general @ref[compatibility guidelines](../compatibility-guidelines.md).
+
+## Akka HTTP 10.1.11 - > 10.2.0
```

Comment:
If we backport this fully, we need to adapt this. Depending on whether we release 10.1.12 first or 10.2.0, it probably suffices to add this section only once.